source/draw/Map.ooc

```diff
@@ -53,22 +53,13 @@ _Map: class {
 }
 
 Map: abstract class {
-	model: FloatTransform3D { get set }
-	view: FloatTransform3D { get set }
-	projection: FloatTransform3D { get set }
-	textureTransform: FloatTransform3D { get set }
```

Comment:
These variables are global side effects between all modules using the same shaders.